hack/lib/mesh_resources/networkpolicy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
Should we ship this together with Knative Serving maybe? Or are we going to document that users have to apply this manually?

Reply:
I think that it should be alright to ship it with Knative Serving (When istio ingress class is enabled, the networkpolicy is applied.).
@skonto are you alright? I think we need to update the doc https://docs.openshift.com/container-platform/4.6/serverless/serverless-release-notes.html#serverless-rn-1-16-0_serverless-release-notes as well.